### PR TITLE
更新UI显示效果;RoomInfo中增加net_error

### PR DESCRIPTION
**Pull request by Toxiad**
更新UI显示效果;
RoomInfo中增加net_error;

### CODE REVIEW

File: BiLiRoku/About.xaml [hunk 1]
```diff
@@ -18,14 +18,17 @@
     </Window.CommandBindings>
     <Grid>
         <Grid.ColumnDefinitions>
-            <ColumnDefinition Width="129*"/>
-            <ColumnDefinition Width="335*"/>
```
Comment (zyzsdy) [inline BiLiRoku/About.xaml:22]: I don't know what are you dissatisfied with ratio 129: 335, but this is implemented by design.

File: BiLiRoku/About.xaml.cs [hunk 2]
```diff
@@ -30,5 +30,10 @@ private void Window_Loaded(object sender, RoutedEventArgs e)
             InfoBox.Text = "HomePage: https://zyzsdy.com/biliroku\n\n更新说明：\n" + Ver.DESC + "\n\n";
             InfoBox.AppendText("注意：\n本程序不是bilibili官方出品。\n请在不违反bilibili用户协议的前提下使用。\n请遵守直播礼仪，未经up主同意请勿上传直播录像。\n\nBiliRoku可能会收集一些您的设备信息用于改进程序，其中不包含您的任何隐私信息。");
         }
+
+        private void Report_Click(object sender, RoutedEventArgs e)
+        {
+            System.Diagnostics.Process.Start("explorer.exe", "https://github.com/zyzsdy/biliroku");
+        }
```
Comment (zyzsdy) [inline BiLiRoku/About.xaml.cs:37]: If I really need a feedback button, I guess I would jump directly to the Github Issue.

**Review by zyzsdy — CHANGES_REQUESTED** [review]
Currently, Biliroku does **NOT** accept pull requests for UI. Could you please consider withdrawing the changes of UI part.

**Comment by Toxiad** [comment]
> Currently, Biliroku does **NOT** accept pull requests for UI. Could you please consider withdrawing the changes of UI part.

那可以选择 #52

**Pull request closed without merging**